meta-luneos/recipes-webos/nyx-conf/nyx-conf/pinephonepro/nyx.conf

```diff
@@ -0,0 +1,2 @@
+[module.keys]
+paths=/dev/input/event1;/dev/input/event4
```

Comment:
we should try to point to sys bus instead, to have more stable paths; but that can be done later